### ISSUE TITLE
Mock JSON for users

### ISSUE THREAD

**Issue by angularganga** [codaga/tranquil#10]
user id
user name
address1
address2
city
country
postal code
phone1
phone2
email
specialty

**Comment by davisokoth** [comment]
Am not very sure about gender for this, would there be any harm in skipping that? And I imagine `phone`, `email` and `specialty` records could be in multiples so I was wondering if we could have them as arrays of objects? And we could also add a separate name for the clinic or practice Something kinda like this:
`{
  user_id: 100,
  name: 'mimi',
  address: 'Thousand oaks, Luna',
  city: 'Talent Haven',
  country: 'Arctic',
  postal_code: '01320',
  phone: ['1029009090909', '909043092989'],
  email: ['info@minded.com', 'dramit@minded.com'],
  specialty: ['psychiatrist'],
  isactive: true
}`

**Comment by angularganga** [comment]
Yes, we can skip the gender if you prefer.  We can go with your recommendation to make them as arrays.
